airflow/config_templates/config.yml

```diff
@@ -870,6 +870,16 @@ logging:
       type: string
       example: "0o664"
       default: "0o664"
+    reset_and_split_logging:
```

Comment:
This should somehow indicate that it is about Celery logs.

Reply:
What do you think about:
`celery_logging_override`

Reply:
I updated it to *_split. 

Let's wait for additional confirmation before resolving this topic.